apollo-router/src/plugins/telemetry/metrics/otlp.rs

```diff
@@ -63,6 +67,23 @@ impl MetricsConfigurator for super::super::otlp::Config {
                             .with_timeout(self.batch_processor.max_export_timeout)
                             .build(),
                     );
+                if let Some(custom_buckets) = metrics_config.buckets.get_custom() {
+                    for (instrument_name, buckets) in custom_buckets {
+                        let name: &'static str =
+                            Box::leak(instrument_name.clone().into_boxed_str());
+                        // Add this here to be able to clean it once we reload the MeterProvider
+                        builder.instrument_names_to_clean.push(name);
+                        let view = new_view(
+                            Instrument::new().name(name),
```

Comment:
I mentioned it in slack, but adding context here too: the `name()` method takes a `Cow<'static, str>` as argument, but we don't have to jump through hoops to make a static str, since `Cow` implements `From<String>` (which will make a `Cow::Owned` directly